core/src/main/java/org/polypheny/db/algebra/core/lpg/LpgValues.java

```diff
@@ -57,7 +57,10 @@ public NodeType getNodeType() {
 
     @Override
     public String algCompareString() {
-        return "$" + getClass().getSimpleName() + "$" + nodes.hashCode() + "$" + edges.hashCode() + "$" + values.hashCode();
+        return getClass().getSimpleName() + "$"
+                + nodes.hashCode() + "$"
+                + edges.hashCode() + "$"
+                + values.hashCode() + "&";
```

Comment:
Is it necessary for the values to be part of the alg compare string, since the structure, i.e. the nodes and edges, are part of it?

Reply:
Yes, as you can still create a non-graph value for querying, per cypher standard e.g. UNWIND [1, 2, 3, null] AS x
RETURN x, 'val' AS y